hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/record/AbstractNearCacheRecord.java

```diff
@@ -183,9 +172,9 @@ public String toString() {
                 + ", sequence=" + sequence
                 + ", uuid=" + uuid
                 + ", expirationTime=" + expirationTime
-                + ", accessTime=" + accessTime
-                + ", accessHit=" + accessHit
-                + ", recordState=" + recordState
+                + ", lastAccessTime=" + lastAccessTime
+                + ", hits=" + hits
+                + ", recordState=" + reservationId
```

Comment:
Shall we include all new fields like `invalidateNearCache` to the `toString` method?

Reply:
added all fields.